simtools/visualization/legend_handlers.py

```diff
@@ -5,7 +5,7 @@
 from simtools.corsika.corsika_config import CorsikaConfig
 from simtools.io_handler import IOHandler
 from simtools.utils import names
-from simtools.utils.names import lst, mst, sct, sst
+from simtools.utils.names import hess, lst, magic, mst, sct, sst, veritas
```

Comment:
Really thing that this is not good. Could we open an issue to rethink the handling of telescope types in names.py? Best to decouple this issue from this pull request.

Reply:
We could probably modify this `import` to import the dictionary that contains the telescope names, but then we would need additional changes in many other places, which I don't think would make the code more "legible".

Specifically this implementation in `names.py` of the explicit names of the telescopes was done by @VictorBarbosaMartins. At the time he convinced me that it's the best solution, so I trust it is. Perhaps @VictorBarbosaMartins can explain better.

Reply:
OK, I think this is fine for this PR and we should merge.

Do you want to open an issue that we revisit this and optimise it? I think this cannot be the best approach, I don't understand we can't have a variable/list "telescope_types" and simply loop? 

As said, let's decouple this from this PR so that we can move forward.

Reply:
> We could probably modify this `import` to import the dictionary that contains the telescope names, but then we would need additional changes in many other places, which I don't think would make the code more "legible".
> 
> Specifically this implementation in `names.py` of the explicit names of the telescopes was done by @VictorBarbosaMartins. At the time he convinced me that it's the best solution, so I trust it is. Perhaps @VictorBarbosaMartins can explain better.

This was the solution we found to avoid writing the telescope names as strings throughout the code, after @GernotMaier's commented on the previous implementation. This way, the telescope names are defined once, imported and used as an object instead of a string, which is more elegant and centralized. A loop through all the telescope name types will not always serve our purposes, because in several situations we do need to name the telescope type explicitly, and in that case we thought it is better to name it through the object `mst` instead of the non-centralized string `"mst"`. Please feel free to rethink it and make other suggestions.